desktop/cmp/dock/impl/Dock.scss

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+
+.xh-dock-container {
+  position: absolute !important;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  pointer-events: none;
+  align-items: flex-end;
+  justify-content: flex-end;
+
+  &.xh-dock-container-ltr {
```

Comment:
I dunno how you feel about the BEM style or if you want to keep at that - I know it's awkward, but maybe adds some value.  I am not going to advocate strongly for it at this point, but if we wanted to use it there are a few pretty clear cut cases here where we have the sub-elements (header/body) and modifiers (ltr, collapsed)